TreeTracker/UI/Views/Profile Screen/ProfileViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ProfileViewController.swift
+//  TreeTracker
+//
+//  Created by Remi Varghese on 3/23/21.
+//  Copyright © 2021 Greenstand. All rights reserved.
+//
+
+import UIKit
+
+class ProfileViewController: UIViewController {
+    var trees: [Tree] = []
+    weak var rootcoordinator: RootCoordinator!
+    var homecoordinator: HomeCoordinator!
+    // MARK: - IBOutlets
+    @IBOutlet weak var profilepic: UIImageView!
+    @IBOutlet private var nameLabel: UILabel! {
+        didSet {
+            nameLabel.font = FontFamily.Montserrat.semiBold.font(size: 16.0)
+        }
+    }
+    @IBOutlet private var emailphoneLabel: UILabel! {
```

Comment:
How about `usernameLabel` for the variable name?

Reply:
I changed the label name as usernameLabel